iphone/Maps/Classes/MapViewController.mm

```diff
@@ -579,22 +579,19 @@ - (void)processMyPositionStateModeEvent:(location::EMyPositionMode)mode
 
   switch (mode)
   {
-    case location::MODE_UNKNOWN_POSITION:
-    {
+    case location::PendingPosition:
       self.disableStandbyOnLocationStateMode = NO;
-      if (![Alohalytics isFirstSession])
-        [[MapsAppDelegate theApp].locationManager stop:self];
+      [[MapsAppDelegate theApp].locationManager start:self];
       break;
-    }
-    case location::MODE_PENDING_POSITION:
+    case location::NotFollowNoPosition:
+      //TODO(iOS team): show dialog if it is not first launch
```

Comment:
Давай сформулирую вопрос по-другому: в каких именно ситуациях и после каких действий будет выполняться эта строчка кода?

> On Apr 14, 2016, at 13:28, Roman Kuznetsov <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/MapViewController.mm <https://github.com/mapsme/omim/pull/2881#discussion_r59694409>:
> 
> >        break;
> > -    }
> > -    case location::MODE_PENDING_POSITION:
> > +    case location::NotFollowNoPosition:
> > +      //TODO(iOS team): show dialog if it is not first launch
> Разница должна быть :) Потому как данный реквест не имеет никакого отношения к тому, о чем ты спрашиваешь. Я понятие не имею, как часто это будет показываться и что там будет написано. Это решает Игорь, и это другая задача. Давай не засорять данный PR сторонними вещами.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2881/files/0ae1ed3e775cc8d766c830f0702401275c08a573#r59694409>